### ISSUE TITLE
Cannot read runtime data.

### ISSUE THREAD

**Issue by namankumar** [automl/auto-sklearn#193]
Hi there, 

So I have successfully installed auto-sklearn on a 2015 Macbook Air running Mac OS X 10.12.1. During execution, however, my installation cannot read runtime data. Looks like the error is thrown at [this line](https://github.com/automl/auto-sklearn/blob/master/autosklearn/smbo.py#L732). 

I'm attaching my code and the cmd output for the boston dataset. Would appreciate all help in getting autosklearn up and running asap.

Thanks!

```
import sklearn.cross_validation
import sklearn.datasets
import sklearn.metrics

import autosklearn.regression

boston = sklearn.datasets.load_boston()
X = boston.data
y = boston.target


X_train, X_test, y_train, y_test = sklearn.cross_validation.train_test_split(X, y, random\
_state=1)

automl = autosklearn.regression.AutoSklearnRegressor(
    time_left_for_this_task=120, per_run_time_limit=30,
    tmp_folder='/Users/xyz/Downloads/data/temp',
    output_folder='/Users/xyz/Downloads/data/output')

automl.fit(X_train, y_train, dataset_name='boston')

print(automl.show_models())
predictions = automl.predict(X_test)
print("Final R2 score:", sklearn.metrics.r2_score(y_test, predictions))

```

Output: (see very first line for the culprit)
> **[CRITICAL] [2016-11-21 12:09:48,622:AutoMLSMBO(1)::boston] Cannot read runtime data.**
> You are already timing task: index_run4
> You are already timing task: index_run8
> You are already timing task: index_run8
> You are already timing task: index_run8
> You are already timing task: index_run8
> You are already timing task: index_run9
> You are already timing task: index_run11
> You are already timing task: index_run11
> You are already timing task: index_run12
> You are already timing task: index_run12
> You are already timing task: index_run13
> [(0.400000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'median',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'no_preprocessing',
>   'regressor:__choice__': 'random_forest',
>   'regressor:random_forest:bootstrap': 'False',
>   'regressor:random_forest:criterion': 'mse',
>   'regressor:random_forest:max_depth': 'None',
>   'regressor:random_forest:max_features': 3.463799888732291,
>   'regressor:random_forest:max_leaf_nodes': 'None',
>   'regressor:random_forest:min_samples_leaf': 1,
>   'regressor:random_forest:min_samples_split': 3,
>   'regressor:random_forest:min_weight_fraction_leaf': 0.0,
>   'regressor:random_forest:n_estimators': 100,
>   'rescaling:__choice__': 'min/max'})),
> (0.300000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'most_frequent',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'no_preprocessing',
>   'regressor:__choice__': 'gaussian_process',
>   'regressor:gaussian_process:nugget': 0.002928360340513877,
>   'regressor:gaussian_process:thetaL': 2.996376577404654e-06,
>   'regressor:gaussian_process:thetaU': 0.24503015021731772,
>   'rescaling:__choice__': 'min/max'})),
> (0.220000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'mean',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'polynomial',
>   'preprocessor:polynomial:degree': 2,
>   'preprocessor:polynomial:include_bias': 'True',
>   'preprocessor:polynomial:interaction_only': 'True',
>   'regressor:__choice__': 'liblinear_svr',
>   'regressor:liblinear_svr:C': 15481.425760646953,
>   'regressor:liblinear_svr:dual': 'False',
>   'regressor:liblinear_svr:epsilon': 0.0017223387418553921,
>   'regressor:liblinear_svr:fit_intercept': 'True',
>   'regressor:liblinear_svr:intercept_scaling': 1,
>   'regressor:liblinear_svr:loss': 'squared_epsilon_insensitive',
>   'regressor:liblinear_svr:tol': 0.00024043049268107142,
>   'rescaling:__choice__': 'min/max'})),
> (0.040000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'most_frequent',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'no_preprocessing',
>   'regressor:__choice__': 'gaussian_process',
>   'regressor:gaussian_process:nugget': 0.015444025222872566,
>   'regressor:gaussian_process:thetaL': 0.00010000000000000009,
>   'regressor:gaussian_process:thetaU': 1.0,
>   'rescaling:__choice__': 'min/max'})),
> (0.020000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'most_frequent',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'nystroem_sampler',
>   'preprocessor:nystroem_sampler:gamma': 4.230503814139951,
>   'preprocessor:nystroem_sampler:kernel': 'rbf',
>   'preprocessor:nystroem_sampler:n_components': 4020,
>   'regressor:__choice__': 'sgd',
>   'regressor:sgd:alpha': 6.837765185010804e-06,
>   'regressor:sgd:average': 'True',
>   'regressor:sgd:epsilon': 0.0049024667855225995,
>   'regressor:sgd:eta0': 0.08411030990142922,
>   'regressor:sgd:fit_intercept': 'True',
>   'regressor:sgd:l1_ratio': 5.097720720139073e-07,
>   'regressor:sgd:learning_rate': 'constant',
>   'regressor:sgd:loss': 'epsilon_insensitive',
>   'regressor:sgd:n_iter': 711,
>   'regressor:sgd:penalty': 'elasticnet',
>   'rescaling:__choice__': 'min/max'})),
> (0.020000, SimpleRegressionPipeline(configuration={
>   'imputation:strategy': 'mean',
>   'one_hot_encoding:use_minimum_fraction': 'False',
>   'preprocessor:__choice__': 'nystroem_sampler',
>   'preprocessor:nystroem_sampler:coef0': 0.1566448365058719,
>   'preprocessor:nystroem_sampler:degree': 5,
>   'preprocessor:nystroem_sampler:gamma': 0.7004476388937543,
>   'preprocessor:nystroem_sampler:kernel': 'poly',
>   'preprocessor:nystroem_sampler:n_components': 956,
>   'regressor:__choice__': 'ridge_regression',
>   'regressor:ridge_regression:alpha': 0.02849145741142527,
>   'regressor:ridge_regression:fit_intercept': 'True',
>   'regressor:ridge_regression:tol': 0.001397013116831936,
>   'rescaling:__choice__': 'min/max'})),
> ]
> Final R2 score: 0.924461243278
>

**Comment by mfeurer** [comment]
Thanks for reporting this. While this is a critical message, it actually does not affect the performance of auto-sklearn and you can ignore it. Therefore, the error message won't be in the next release of auto-sklearn.

**Comment by namankumar** [comment]
Ok thanks so much. I am also getting these two lines in my log. Would you please confirm if these pose a problem. I can log a new issue if these are significant errors. Thanks.

```
capi_return is NULL
Call-back cb_calcfc_in__cobyla__user__routines failed.

```

**Comment by mfeurer** [comment]
This looks like an issue with FastICA. Anyway, calling machine learning algorithms is wrapped in a try/except, so this error won't make auto-sklearn fail.

**Comment by namankumar** [comment]
Ok, thanks. I can debug my FastICA installation.

**Comment by mfeurer** [comment]
There is no need to debug a target algorithm. Unless this happens over and over again, it is most likely that the preprocessing (especially if your data contains categorical features) makes the algorithm later in the pipeline fail.

**Comment by namankumar** [comment]
It happens more than a few times though not every time. What you say makes sense, I'll skip over this issue for now. Thanks!
